packages/editor/src/components/collab-sidebar/index.js

```diff
@@ -288,6 +290,11 @@ export default function CollabSidebar() {
 			}
 		} );
 
+		result.map( ( item ) => {
+			item.reply = item.reply.reverse();
```

Comment:
Maybe we should clone the array first?

Reply:
You are right, we shouldn't mutate the original object.